.github/workflows/ci.yml

```diff
@@ -1,8 +1,10 @@
 # https://help.github.com/en/categories/automating-your-workflow-with-github-actions
 
 on:
-  pull_request:
   push:
+    branches:
+      - main
```

Comment:
This should currently be `master`.  The repo hasn't had the anti-slavery rename yet.

Reply:
Done.